esmvaltool/recipes/recipe_enso_basicclimatology.yml

```diff
@@ -0,0 +1,185 @@
+# ESMValTool enso basic climatology
+#
+---
+documentation:
+  description: ENSO CLIVAR metrics, background climatology, by Yann Planton
+  title: Reproducing ENSO CLIVAR metrics - background climatology
+  authors:
+    - chun_felicity
+    - beucher_romain
+    - planton_yann
+  maintainer:
+    - chun_felicity
+
+datasets:
+  ## test run with subset of CMIP6 models ##
+  - {dataset: ACCESS-ESM1-5, project: CMIP6, exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 1850, end_year: 2014}
```

Comment:
Could we also have here only one model listed? The REF will intercompare each model individually with the observations.